pmd-java/src/main/resources/category/java/bestpractices.xml

```diff
@@ -1831,7 +1831,49 @@ public class Foo {
         </example>
     </rule>
 
-    <rule name="UseTryWithResources"
+  <rule name="UseStandardCharsets"
+        language="java"
+        since="6.34"
+        minimumLanguageVersion="1.7"
+        message="Please use StandardCharsets constants"
+        class="net.sourceforge.pmd.lang.rule.XPathRule"
+        externalInfoUrl="${pmd.website.baseurl}/pmd_rules_java_bestpractices.html#usestandardcharsets">
+    <description>
+      StandardCharsets has constants for various common character
+      sets, they should be used instead of calling &quot;Charset.forName(name)&quot;
+    </description>
```

Comment:
I think you did a good job of motivating the rule in #3190, and this should be part of the description.

Btw you can use markdown in the descriptions, for instance use backticks (`` ` ``) instead of `&quot;`, which is better to format code: `` `Charset.forName(name)` ``

Reply:
Also I think your xml formatter uses 2 spaces indentation, but this file uses 4 spaces